logs/plugin_bot.go

```diff
@@ -124,13 +124,13 @@ var cmdWhois = &commands.YAGCommand{
 		}
 
 		var memberStatus string
+		state := [6]string{"Playing", "Streaming", "Listening", "Watching", "Custom", "Competing"}
 		if !member.PresenceSet || member.PresenceGame == nil {
-			memberStatus = fmt.Sprintf("Has no active status, is invisible/offline or is not in the bot's cache.")
+			memberStatus = "Has no active status, is invisible/offline or is not in the bot's cache."
 		} else {
 			if member.PresenceGame.Type == 4 {
 				memberStatus = fmt.Sprintf("%s: %s", member.PresenceGame.Name, member.PresenceGame.State)
 			} else {
-				state := [4]string{"Playing", "Streaming", "Listening", "Watching"}
 				presenceName := "Unknown"
 				if member.PresenceGame.Type > 0 && len(state) > int(member.PresenceGame.Type) {
```

Comment:
i spotted a bug i left in here, should be `member.PresenceGame.Type >= 0` unless im wrong

Reply:
Yup, sounds like you're right.